lib/matplotlib/colors.py

```diff
@@ -960,6 +960,229 @@ def scaled(self):
         return (self.vmin is not None and self.vmax is not None)
 
 
+class FuncNorm(Normalize):
+    """
+    Creates a normalizer using a custom function
+
+    The normalizer will be a function mapping the data values into colormap
+    values in the [0,1] range.
+    """
+
+    def __init__(self, f, finv=None, **normalize_kw):
```

Comment:
I understand that kwargs is what is used as the general case, however, I used normalize_kw because all of these parameters are to be passed to the parent class `Normalize`. This is the same name convention used for [subplots](https://github.com/matplotlib/matplotlib/blob/409d8b79074f99a2fd5b0e2498aab24535e40485/lib/matplotlib/figure.py#L1027).